client/connector_token_create.go

```diff
@@ -125,7 +133,11 @@ func (cli *VanClient) ConnectorTokenCreateFile(ctx context.Context, subject stri
 	if !res.Allowed {
 		return res.Err()
 	}
-	secret, localOnly, err := cli.ConnectorTokenCreate(ctx, subject, "")
+	siteConfig, err := cli.getSiteConfig(ctx, cli.Namespace)
+	if err != nil {
+		return err
+	}
+	secret, localOnly, err := cli.ConnectorTokenCreate(ctx, fmt.Sprintf("%s-%s", subject, siteConfig.Reference.UID), "")
```

Comment:
Ideally we would separate the name used in the secret yaml from the subject used in the client certificate within that secret.

Reply:
LGTM. I can move the concatenation of subject and site UID into `ConnectorTokenCreate`, but the function is called at several points in the code (e.g. `newGateway`  and `GatewayGenerateBundle`). The alternative could be to change the signature of the function.